.github/workflows/nightly.yml

```diff
@@ -27,13 +35,14 @@ jobs:
         python setup.py bdist_wheel sdist
     - name: Test Packages
       run: |
-        pip install $(ls dist/*.whl)
+        pip install $(ls dist/*.whl) --ignore-installed
         allennlp test-install
     - name: Check diff from 24hr ago
       if: success()
       run: scripts/24hr_diff.sh
     - name: Deploy
-      if: success()
+      # Only run if the previous steps succeeded and we are on the upstream git repo.
+      if: success() && github.repository == 'allenai/allennlp'
```

Comment:
Maybe you can check this earlier in the script, so nothing is run if it's a fork? See [this answer](https://github.community/t5/GitHub-Actions/don-t-run-actions-on-pull-request-from-fork/td-p/45499) for example.